### PR TITLE
package upgrades

### PR DESCRIPTION
**Pull request by PavelFilo**
maintenance of packages

### CODE REVIEW

File: package.json
Comment (radoslavzeman) [inline package.json:1]: Please keep all `dependencies` on fixed version or at least on fixed minor version (using `~`, not `^`)

File: utils/paymentRedirect.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ export const paymentRedirect = (ticketInit: TicketInitDto, paymentOption: Paymen
       paymentOption === 'apple-pay'
         ? ticketInit.paymentUrls.paymentUrlAPAY
         : paymentOption === 'google-pay'
-        ? ticketInit.paymentUrls.paymentUrlGPAY
-        : ticketInit.paymentUrls.paymentUrlCard
+          ? ticketInit.paymentUrls.paymentUrlGPAY
+          : ticketInit.paymentUrls.paymentUrlCard
```
Comment (radoslavzeman) [inline utils/paymentRedirect.ts:20]: Just checking formatting, I think this is not forced by prettier, but by you IDE and it should not be. (Not a blocker for merging)

**Review by radoslavzeman — APPROVED** [review]
🙌

File: utils/paymentRedirect.ts (hunk 1 above)
Comment (PavelFilo) [reply utils/paymentRedirect.ts:20]: this was an error in ts:check

**Pull request closed without merging**